src/test/java/organice/logic/commands/MatchCommandTest.java

```diff
@@ -245,6 +245,11 @@ public Patient getPatient(Nric patientNric) {
             throw new AssertionError("This method should not be called.");
         }
 
+        @Override
```

Comment:
I remember making a PR that creates a public ModelStub class, so we don't have to change the MatchCommandTest and AddCommandTest's ModelStub classes.

Reply:
Is it merged? Because I need to change those things if not the code cnt pass the tests

Reply:
not sure but ytd @Yen-Peng error also because of this override

Reply:
should be fixed